scripts/testbucket-project-store.py

```
@@ -0,0 +1,131 @@
+
+
+def fit(low, high, lo_c, hi_c, to_c):
+    """Two-point fit, clamped: elapsed(c) = fixed + c*per_iter."""
+    per = max((high - low) / float(hi_c - lo_c), 0.0)
+    fixed = max(low - per * lo_c, 0.0)
+    return fixed + to_c * per
+
+
+def main():
+    ap = argparse.ArgumentParser()
+    ap.add_argument("--low", required=True); ap.add_argument("--low-count", type=int, required=True)
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)